addons/modules/stringtable.xml

```diff
@@ -2244,5 +2244,14 @@
             <English>Unable to find suitable position to teleport vehicle to.</English>
             <German>Kein valide Position zum Teleporieren des Fahrzeugs gefunden.</German>
         </Key>
+        <Key ID="STR_ZEN_Modules_ModuleCrewToGunner">
+            <English>Crew To Gunner</English>
+        </Key>
```

Comment:
Just the feedback messages here. Move the module name up to be with the rest.

Reply:
https://github.com/zen-mod/ZEN/pull/423/commits/4720addf040fbd3cbf43e9b8f79f8a13cddca2a3